pkgs/applications/misc/qwalkingpad/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv
+, lib
+, qtbase
+, qtconnectivity
+, wrapQtAppsHook
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "qwalkingpad";
+  version = "0.1-git.556bca7";
+
+  src = fetchFromGitHub {
+    owner = "DorianRudolph";
+    repo = "QWalkingPad";
+    rev = "556bca7e610c863aab33f9b8d66f98b574272fac";
```

Comment:
Note that this revision hasn't been accepted into their repository just yet: https://github.com/DorianRudolph/QWalkingPad/pull/7. Maybe we should wait for that.